<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendSave</class>
 <widget class="QDialog" name="SendSave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>发送列表</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QToolButton" name="send2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="send3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="send">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="add">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QToolButton" name="clear">
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="send1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QTableWidget" name="tbSave"/>
   </item>
   <item row="1" column="4">
    <widget class="QToolButton" name="show">
     <property name="text">
      <string>记录</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
